x-pack/test/security_solution_cypress/cypress/e2e/detection_response/rule_management/prebuilt_rules/prebuilt_rules_preview.cy.ts

```diff
@@ -220,9 +220,13 @@ describe('Detection rules, Prebuilt Rules Installation and Update workflow', ()
       type: 'machine_learning',
       anomaly_threshold: 65,
       machine_learning_job_id: ['auth_high_count_logon_events', 'auth_high_count_logon_fails'],
+      alert_suppression: undefined,
     }),
     ['security-rule.query', 'security-rule.language']
```

Comment:
nit: for ES|QL suppression I added tests for use case when alert_suppression is not `undefined`
https://github.com/elastic/kibana/pull/180927/commits/8702a4862ca910b459ebbd0a600bc297e22a8628

Reply:
A +1 to adding an assertion in the `Machine learning rule properties` test to `assertAlertSuppressionPropertiesShown`

Reply:
https://github.com/elastic/kibana/pull/181926/commits/96141dd15e2e7ad9c7271fcfde008d32ae201efd